content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -395,6 +393,9 @@ different Kubernetes components.
 | `MountPropagation` | `true` | GA | 1.12 | - |
 | `NamespaceDefaultLabelName` | `true` | Beta | 1.21 | 1.21 |
 | `NamespaceDefaultLabelName` | `true` | GA | 1.22 | - |
+| `NetworkPolicyEndPort` | `false` | Alpha | 1.21 | 1.21 |
+| `NetworkPolicyEndPort` | `true` | Beta | 1.22 |  |
```

Comment:
Don't we wanna add the "until" here for v1.24?

Reply:
Yep, we should add that.